### PR TITLE
#85: register models with admin.py

### PR DESCRIPTION
**Pull request by Nerdenator**
This will enable users to add tags and addresses through the Django admin interface.

NOTE: a superuser will need to be created in the environment. See https://docs.djangoproject.com/en/1.11/ref/django-admin/#django-admin-createsuperuser

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +88,13 @@ $ make stop
    $ make migratelocal
    # or run in detached mode
    $ make && make upd && make migratelocal
+   # to add superuser:
+   # first, use docker ps to get container ID for the web container
+   $ docker ps
+   # next, run the command to execute a Bash shell in the web container
+   $ docker exec -it <your_container_id_here> bash
```
[the remark this answers is not in the document]
Comment (Nerdenator) [reply README.md]: Changed.

File: backend/admin.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
 from django.contrib import admin
-
+from .models import Address, Tag
```
[the remark this answers is not in the document]
Comment (Nerdenator) [reply backend/admin.py]: Changed.

File: backend/models.py [hunk 3]
```diff
@@ -51,6 +56,10 @@ class Tag(base_models.Model):
     tag_words = base_models.CharField(max_length=255, blank=True)
     tag_initials = base_models.CharField(max_length=20, blank=True)
 
+    def __str__(self):
+        return str.format('{} {}',self.address, self.id)
```
[the remark this answers is not in the document]
Comment (Nerdenator) [reply backend/models.py]: Resolved

**Comment by fredlawl** [comment]
Thanks for the PR.

**Pull request merged**